core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -154,18 +155,26 @@ object TestUtils extends Logging {
     servers.map(s => formatAddress(s.config.hostName, s.boundPort(SecurityProtocol.SSL))).mkString(",")
   }
 
+  def getSaslBrokerListStrFromServers(servers: Seq[KafkaServer]): String = {
```

Comment:
Instead of having one method per security protocol, why not just take the security protocol as a parameter? Then 3 methods would become one. The implementation seems exactly the same apart from the security protocol passed to `boundPort`.